### PR TITLE
Wikipedia Covid-19 TimeSeries Import

### PR DESCRIPTION
**Pull request by edumorlom**
image
[Click here to view example page.](https://en.wikipedia.org/w/index.php?title=Template:COVID-19_pandemic_data/United_States_medical_cases_chart)

1. For every URL in input.txt, get the name of the place.
2. Parse the time series table data for that specific URL.
         a. The table rows are identifiable via the color.
         b. We then match the color to its label.
         c. For example: "red"  === "deaths" in image above.
3. Get the WikidataId for that specific place by searching on Wikipedia.
4. Store all the tables.
5. Combine all tables into one big table.
6. Export the big table to CSV. Each row contains WikidataId.

**The main method can be found in wiki_parser.py.**

### CODE REVIEW

File: scripts/wikipedia_covid/timeseries/wiki_parser.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+    # Get a map of color -> label
+    # The table doesn't refer to items by label but rather by color.
+    span_labels = table.findAll("span", {"style": "font-size:90%; margin:0px"})
+
+    color_to_label = {}
+
+    for span_label in span_labels:
+        # There must be two contents inside the span.
+        # conten1: color
```
Comment (edumorlom) [inline scripts/wikipedia_covid/timeseries/wiki_parser.py:107]: typo

File: scripts/wikipedia_covid/timeseries/CovidWikipedia.mcf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Node: dcid:Count_MedicalConditionIncident_COVID_19_ConfirmedCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+medicalStatus: dcs:ConfirmedCase
+medicalCondition: dcs:COVID_19
```
Comment (tjann) [inline scripts/wikipedia_covid/timeseries/CovidWikipedia.mcf:7]: Same note as from other PR re incidentType. BTW these are the same StatVar? If one is submitted first, we can remove the other.

File: scripts/wikipedia_covid/timeseries/Config.py [hunk 3]
```diff
@@ -0,0 +1,1155 @@
+# NOTE: Keep all lower case.
+
+replace_headers = {
+    "active cases": "active_cases", # Get rid of spaces
+    "active cases and recoveries": "cases", # See Connecticut, USA
```
Comment (tjann) [inline scripts/wikipedia_covid/timeseries/Config.py:5]: I'm not sure the "See" comments are helpful-- expand or remove?

File: scripts/wikipedia_covid/timeseries/wiki_parser.py [hunk 4]
```diff
@@ -0,0 +1,239 @@
+import datetime
```
Comment (tjann) [inline scripts/wikipedia_covid/timeseries/wiki_parser.py:1]: @edumorlom  can you modularize this and add tests? I'll review after!

**Review by tjann — CHANGES_REQUESTED** [review]

File: scripts/wikipedia_covid/timeseries/CovidWikipedia.mcf (hunk 2 above)
Comment (edumorlom) [reply scripts/wikipedia_covid/timeseries/CovidWikipedia.mcf:7]: sounds good

File: scripts/wikipedia_covid/timeseries/Config.py (hunk 3 above)
Comment (edumorlom) [reply scripts/wikipedia_covid/timeseries/Config.py:5]: done

File: scripts/wikipedia_covid/timeseries/wiki_parser.py (hunk 4 above)
Comment (edumorlom) [reply scripts/wikipedia_covid/timeseries/wiki_parser.py]: Done, thanks

**Pull request closed without merging**